go.mod

```diff
@@ -34,3 +34,95 @@ require (
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.20.5
+
+require (
```

Comment:
it's strange to have all these changes, have you correctly rebased on master branch?

Reply:
I did a `pull` and accepted both the changes and ran `go mod tidy`.
What would you suggest I do?